### ISSUE TITLE
Determine container working directory correctly

### ISSUE THREAD

**Issue by reitzig** [reitzig/container-do#19]
**Given**

```Dockerfile
FROM alpine
WORKDIR /work
```

Build with Kaniko!

```toml
[container]
image = "custom-image"
```

**When:**

```
container-do pwd
```

**Expected:** Output `/work`
**Actual:** Output `/`

**Comment by reitzig** [comment]
_Analysis:_ Kaniko doesn't seem to fill `.ContainerConfig(.WorkingDir)` in the same way as `docker`. We were [probably](https://forums.docker.com/t/what-is-the-difference-between-containerconfig-and-config-in-image/83232) wrong in using it in the first place.

--> use `.Config.WorkingDir` instead.

**Comment by reitzig** [comment]
Done; will be released with 1.2.3.
